02-IDE_exercises/solutions/03b-project/find_factors.py

```diff
@@ -2,9 +2,9 @@
 # Date: 10/05/2020
 # Description: Take in positive integer and print list of factors
 
-integer = int(input("Please enter a positive integer:"))
+integer: int = int(input("Please enter a positive integer:"))
 print("The factors of", integer, "are:")
 
-for i in range(1, integer+1):
+for i in range(1, integer + 1):  # type: int
```

Comment:
I looked up the PEP for when you declare variables in loops. Best practice for type hinting is to declare the variable before the loop like so:

```
i: int
for i in range(1, integer +1):
```